applications/MultilevelMonteCarloApplication/custom_statistics/power_sums_statistics.cpp

```diff
@@ -88,7 +88,13 @@ namespace Kratos
         KRATOS_TRY;
 
         // Check and set number of elements and check dimension
-        KRATOS_ERROR_IF(mrModelPart.NumberOfNodes() == 0) << "The number of nodes in the domain is zero. The power sums statistic cannot be applied."<< std::endl;
+        bool hasNodes = 0;
```

Comment:
```suggestion
        bool has_nodes = 0;
```
Following style guide

Reply:
Oki